packages/git-encrypted/package.json

```diff
@@ -63,6 +63,7 @@
     "typescript": "^4.1.2"
   },
   "dependencies": {
+    "argon2": "^0.27.1",
```

Comment:
@johannes87 Unfortunately this package is a native node module and requires native code. Therefore it won’t run on pure javascript environments like react-native or the browse, which is a requirement for our git-encryption package. I wonder if there’s another argon2 implementation available which is supported in the browser?